File: 2420/2420.py
# **************************************************************************** #
#                                                                              #
#                                                         :::      ::::::::    #
#    2420.py                                            :+:      :+:    :+:    #
#                                                     +:+ +:+         +:+      #
#    By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+         #
#                                                 +#+#+#+#+#+   +#+            #
#    Created: 2024/03/02 17:12:40 by jihoolee          #+#    #+#              #
#    Updated: 2024/03/02 17:13:20 by jihoolee         ###   ########.fr        #
#                                                                              #
# **************************************************************************** #

def main() -> None:
    n, m = map(int, input().split())
    print(abs(n - m))


if __name__ == '__main__':
    main()

File: 2420/2420.cpp
/* ************************************************************************** */
/*                                                                            */
/*                                                        :::      ::::::::   */
/*   2420.cpp                                           :+:      :+:    :+:   */
/*                                                    +:+ +:+         +:+     */
/*   By: jihoolee <jihoolee@student.42SEOUL.kr>     +#+  +:+       +#+        */
/*                                                +#+#+#+#+#+   +#+           */
/*   Created: 2024/03/02 17:09:19 by jihoolee          #+#    #+#             */
/*   Updated: 2024/03/02 17:12:13 by jihoolee         ###   ########.fr       */
/*                                                                            */
/* ************************************************************************** */

#include <iostream>

int main(void) {
  int64_t  n;
  int64_t  m;

  std::cin >> n >> m;
  if (n > m)
    std::cout << n - m;
  else
    std::cout << m - n;
  return 0;
}
